### PR TITLE
Refactor: use defaults as in prod.Dockerfile for smoketests

### PR DESCRIPTION
**Pull request by nichtsfrei**

### CODE REVIEW

File: smoketest/cmd/scans/main.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ const protocoll = "unix"
 func init() {
 	address = os.Getenv("OSPD_SOCKET")
 	if address == "" {
-		address = "/var/run/ospd/ospd.sock"
+		address = "/var/run/ospd/ospd-openvas.sock"
```
Comment (bjoernricks) [inline smoketest/cmd/scans/main.go:25]:
```suggestion
		address = "/run/ospd/ospd-openvas.sock"
```

File: smoketest/cmd/test/main.go [hunk 2]
```diff
@@ -25,7 +25,7 @@ const protocoll = "unix"
 func init() {
 	ospdSocket = os.Getenv("OSPD_SOCKET")
 	if ospdSocket == "" {
-		ospdSocket = "/var/run/ospd/ospd.sock"
+		ospdSocket = "/var/run/ospd/ospd-openvas.sock"
```
Comment (bjoernricks) [inline smoketest/cmd/test/main.go:28]:
```suggestion
		ospdSocket = "/run/ospd/ospd-openvas.sock"
```

File: smoketest/run-tests.sh [hunk 3]
```diff
@@ -15,15 +15,15 @@ mosquitto -c /etc/mosquitto.conf &
 redis-server /etc/redis/redis.conf
 /usr/sbin/sshd
 ospd-openvas --disable-notus-hashsum-verification True \
-  -u /var/run/ospd/ospd.sock \
+  -u /var/run/ospd/ospd-openvas.sock \
```
Comment (bjoernricks) [inline smoketest/run-tests.sh:18]:
```suggestion
  -u /run/ospd/ospd-openvas.sock \
```

File: smoketest/run-tests.sh [hunk 4]
```diff
@@ -15,15 +15,15 @@ mosquitto -c /etc/mosquitto.conf &
...
   -l /var/log/gvm/ospd.log
 wait_turn=0
-while [ ! -S /var/run/ospd/ospd.sock ]; do
+while [ ! -S /var/run/ospd/ospd-openvas.sock ]; do
```
Comment (bjoernricks) [inline smoketest/run-tests.sh:21]:
```suggestion
while [ ! -S /run/ospd/ospd-openvas.sock ]; do
```

**Review by bjoernricks — COMMENTED** [review]
`/var/run` is actually only a symlink to `/run`. Btw. the default path in https://github.com/greenbone/ospd-openvas/blob/main/ospd/parser.py#L35 should be changed too now since ospd is part of ospd-openvas.

File: smoketest/cmd/scans/main.go (hunk 1 above)
Comment (nichtsfrei) [reply smoketest/cmd/scans/main.go]: I used a sed command to change every case instead of using suggestion; I hope you don't mind.

**Review by ArnoStiefvater — APPROVED** [review]

**Pull request merged**